client/src/main/com/sinch/sdk/domains/conversation/models/v1/credentials/ConversationChannelCredentialsBuilderFactory.java

```diff
@@ -2,90 +2,191 @@
 
 import com.sinch.sdk.domains.conversation.models.v1.ConversationChannel;
 
+/** Factory helper for App credentials builder */
 public class ConversationChannelCredentialsBuilderFactory {
 
+  private ConversationChannelCredentialsBuilderFactory() {}
+
+  /**
+   * Create dedicated conversation credential builder for <code>{@link ConversationChannel#APPLEBC}
+   * </code> channel
+   *
+   * @param credentials Credentials
+   * @return Credentials builder
```

Comment:
Review classes here too:
 - @param:  Credentials -> AppleBusinessChatCredentials
 - @return: Credentials -> ConversationChannelCredentials

Reply:
Same as previous comment: mix between parameter name and used type.
Changed to have better doc
![image](https://github.com/user-attachments/assets/30818217-850b-45ae-9e27-76f6574e332b)

Reply:
Thanks. Too much jsdoc recently on my side 😉